Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -113,6 +113,22 @@ extension MongoClient {
 
         return true
     }
+
+    internal convenience init(options: ClientOptions? = nil) throws {
+        try self.init(connectionString: getConnStr(), options: options)
+    }
+
+    internal convenience init() throws {
```

Comment:
Do we need both of these? I'd think the first method with a default value for options would cover both. 

Also, have you confirmed somehow that the inits here take priority over the ones initially defined in the `MongoClient` class?

Reply:
You're right, I've removed the second `init()`.

Yes, I've confirmed this. I did this by examining the original `init()` signatures:
```
init(connectionString: String = "mongodb://localhost:27017", options: ClientOptions? = nil)
init(fromPointer: OpaquePointer)
```
The second of these can't be used because the signature is different. The first, since it has default values for connectionString, may end up being called. However, convenience initializers are called first. This is testable by trying to run the tests under a non-default connection string and non-default `mongod` port. If the original initializer is called, then the tests should fail to connect to a `mongod`.